docs/Upgrade.md

```diff
@@ -0,0 +1,38 @@
+# Splunk Operator Upgrade
+
+Please note upgrading Splunk operator to Version `1.1.0` is a new installation rather than upgrade from current operator. Due to this  older Splunk operator need to be cleaned up before installing the latest version.  The script will help customer in doing these steps. This script expect current namespace where operator is installed , and path to `1.1.0` manifest file. script will follow the below mentioned steps
```

Comment:
Change to "Upgrading the Splunk operator to Version `1.1.0` is a new installation rather than an upgrade from the current operator. The older Splunk operator must be cleaned up before installing the new version. This script helps you to do the cleanup. The script expects the current namespace where the operator is installed and the path to the `1.1.0` manifest file. The script performs the following steps:

Reply:
Fixed